example/windows/example.sln

```diff
@@ -54,24 +54,24 @@ Global
 		Release|x86 = Release|x86
 	EndGlobalSection
 	GlobalSection(ProjectConfigurationPlatforms) = postSolution
-		{3D9C9FEA-EB1D-4808-9917-E1E90FDBDFDC}.Debug|ARM64.ActiveCfg = Debug|ARM64
-		{3D9C9FEA-EB1D-4808-9917-E1E90FDBDFDC}.Debug|ARM64.Build.0 = Debug|ARM64
-		{3D9C9FEA-EB1D-4808-9917-E1E90FDBDFDC}.Debug|ARM64.Deploy.0 = Debug|ARM64
-		{3D9C9FEA-EB1D-4808-9917-E1E90FDBDFDC}.Debug|x64.ActiveCfg = Debug|x64
-		{3D9C9FEA-EB1D-4808-9917-E1E90FDBDFDC}.Debug|x64.Build.0 = Debug|x64
-		{3D9C9FEA-EB1D-4808-9917-E1E90FDBDFDC}.Debug|x64.Deploy.0 = Debug|x64
-		{3D9C9FEA-EB1D-4808-9917-E1E90FDBDFDC}.Debug|x86.ActiveCfg = Debug|Win32
-		{3D9C9FEA-EB1D-4808-9917-E1E90FDBDFDC}.Debug|x86.Build.0 = Debug|Win32
-		{3D9C9FEA-EB1D-4808-9917-E1E90FDBDFDC}.Debug|x86.Deploy.0 = Debug|Win32
-		{3D9C9FEA-EB1D-4808-9917-E1E90FDBDFDC}.Release|ARM64.ActiveCfg = Release|ARM64
-		{3D9C9FEA-EB1D-4808-9917-E1E90FDBDFDC}.Release|ARM64.Build.0 = Release|ARM64
-		{3D9C9FEA-EB1D-4808-9917-E1E90FDBDFDC}.Release|ARM64.Deploy.0 = Release|ARM64
-		{3D9C9FEA-EB1D-4808-9917-E1E90FDBDFDC}.Release|x64.ActiveCfg = Release|x64
-		{3D9C9FEA-EB1D-4808-9917-E1E90FDBDFDC}.Release|x64.Build.0 = Release|x64
-		{3D9C9FEA-EB1D-4808-9917-E1E90FDBDFDC}.Release|x64.Deploy.0 = Release|x64
-		{3D9C9FEA-EB1D-4808-9917-E1E90FDBDFDC}.Release|x86.ActiveCfg = Release|Win32
-		{3D9C9FEA-EB1D-4808-9917-E1E90FDBDFDC}.Release|x86.Build.0 = Release|Win32
-		{3D9C9FEA-EB1D-4808-9917-E1E90FDBDFDC}.Release|x86.Deploy.0 = Release|Win32
+		{9C5CD441-1749-47EC-90E6-3899B5731898}.Debug|ARM64.ActiveCfg = Debug|ARM64
+		{9C5CD441-1749-47EC-90E6-3899B5731898}.Debug|ARM64.Build.0 = Debug|ARM64
+		{9C5CD441-1749-47EC-90E6-3899B5731898}.Debug|ARM64.Deploy.0 = Debug|ARM64
+		{9C5CD441-1749-47EC-90E6-3899B5731898}.Debug|x64.ActiveCfg = Debug|x64
+		{9C5CD441-1749-47EC-90E6-3899B5731898}.Debug|x64.Build.0 = Debug|x64
+		{9C5CD441-1749-47EC-90E6-3899B5731898}.Debug|x64.Deploy.0 = Debug|x64
+		{9C5CD441-1749-47EC-90E6-3899B5731898}.Debug|x86.ActiveCfg = Debug|Win32
+		{9C5CD441-1749-47EC-90E6-3899B5731898}.Debug|x86.Build.0 = Debug|Win32

```

Comment:
all auto generated jazz from Visual Studio?

Reply:
This was actually the result of running the app on windows, it altered things just a touch in response to `yarn example:windows` and then the files were stable on repeated re-runs so it seemed the right thing to do to commit them, so folks trying the example didn't immediately see dirty git tree after running things